style.css

```diff
@@ -0,0 +1,131 @@
+/* Edit */
+
+h1~p {
+  color: black;
+}
+
+h1 {
+  color: black;
+}
+
+/* Varför fungerar inte detta? */
+section~h1 {
+  color: red;
+}
+/*
+Selektorn section~h1 fungerar inte eftersom den
+ specifikt riktar sig mot h1-element som följer 
+ direkt efter ett section-element som en syskonselektor.
+  Detta innebär att selektorn kommer att matcha alla h1-element
+   som är syskon till ett section-element och kommer efter det,
+    vilket inte är en vanlig struktur. 
+    */
+
+body {
+  font: 16px Arial, sans-serif;
+  margin: 0;
+  padding: 0;
+  background-color: #f4f4f4;
```

Comment:
This is picky, but you're using either text values or hex values for your colors interchangeably, I would stick to either or for consistency. Preferably I'd stick to hex out of the two.